### PR TITLE
tests: refine integration test

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Currently we enable debug level log in integration test, which may lead to high IO in Jenkins CI and CI failure

### What is changed and how it works?
1. change dm-worker log level to `info`
2. check print status after dm-worker exits in `print_status` test

### Check List 

Tests 

 - Integration test

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @GregoryIan @csuzhangxc

File: tests/_utils/run_dm_worker [hunk 1]
```diff
@@ -19,6 +19,6 @@ echo "[$(date)] <<<<<< START DM-WORKER on port $port, config: $conf >>>>>>"
 cd $workdir
 $binary -test.coverprofile="$TEST_DIR/cov.$TEST_NAME.worker.$port.$(date +"%s").out" DEVEL \
     --worker-addr=:$port --relay-dir="$workdir/relay_log" \
-    --log-file="$workdir/log/dm-worker.log" -L=debug --config="$conf" \
+    --log-file="$workdir/log/dm-worker.log" -L=info --config="$conf" \
```
Comment (WangXiangUSTC) [inline tests/_utils/run_dm_worker]: maybe we should reduce some debug log...

File: tests/print_status/run.sh [hunk 2]
```diff
@@ -40,39 +62,21 @@ function run() {
...
-    run_sql_file $cur/data/db.increment.sql $MYSQL_HOST1 $MYSQL_PORT1
-    check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+    echo "check load unit print status success"
 
     # check sync unit print status
-    i=0
-    while [ $i -lt 3 ]
-    do
-        event_count=$(grep "receive binlog event with header" $WORK_DIR/worker1/log/dm-worker.log|grep -v relay|grep "XIDEvent"|wc -l)
-        if [ "$event_count" == "92" ]; then
```
Comment (csuzhangxc) [inline tests/print_status/run.sh]: do not check `event_count` anymore?

Comment (amyangfei) [reply tests/print_status/run.sh]: 1. before this pr, check `event_count` ensures dm-worker log flushed to disk, it is not used for data replication guarantee (As data has been checked via sync_diff).
2. after this pr, we first check data consistency, kill dm-worker and then check print_status.

**Pull request merged**